homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+
+const app = express();
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  console.log(req);
+  res.send("hello from backend to frontend!");
+});
+
+app.post("/weather", async (req, res) => {
+  const cityName = req.body.cityName;
+  res.setHeader("Content-Type", "application/json");
+  res.send(cityName);
```

Comment:
Thanks to the registration of `express.json()` line 4, this can be simplified into:

```suggestion
  res.json(cityName);
```